src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -0,0 +1,86 @@
+<br/><br/>
+
+5. Many assets, like DAI or USDC, exist on multiple networks. When transferring crypto tokens, make sure that the other side is using the same network as you are, since these are not interchangeable.
+6. Confirm the transaction and have prepared a small amount of ETH to pay for a transaction fee, the exact amount of the fee changes depending on network congestion. 
+7. Once your transaction is processed, the crypto will show up in the recipient’s account. This might take anywhere from few seconds to few minutes.
+
+## Connecting to projects
+
+Your address represents you on all Ethereum projects. You will not need to register individually on any project. Once you have a wallet, you can connect to any Ethereum project without any additional information. No emails or any other personal information are needed.
+
+1. Visit any project’s website.
+2. Click a button “Open the App” in the menu if the main page is a presentation of the project. 
+3. Once you are in the app click on “Connect”
+
+![Button allowing user to connect to the website with a wallet](./connect1.png)
+
+4. Select your wallet from the offered list. If you can't see your wallet, it may be hidden under “WalletConnect” option.
+
+![Selecting from a list of wallets to connect with](./connect2.png)
+
+5. Confirm a signature request in your wallet to start the connection. It should not cost anything.
+6. That's it! Start using the app. You can find some interesting projects on our [dApps page](/dapps/#explore).
+
+<InfoBanner shouldSpaceBetween emoji=":eyes:">
+  <div>Want to learn more?</div>
+  <ButtonLink to="/guides/">
+    See our other guides
+  </ButtonLink>
+</InfoBanner>
+
+## Frequently asked questions
+
+### If I own an ETH address, do I own the same address on other blockchains?
+
+You can use the same address on all EVM compatible blockchains (if you have the type of wallet with a recovery phrase). This [list](https://chainlist.org/) will show you which blockchains you can use with the same address. Some blockchains like Bitcoin are completely separate and you will need a different address with a different format. If you have a smart contract wallet you should check its product website for more info on what blockchains are allowed.
+
+### Can I use the same address on multiple devices?
+
+Yes you can use the same address on multiple devices. Wallets are technically only an interface to show you your balance and to make transactions, your account isn't stored inside the wallet, but on the blockchain.
+
+### I have not received the crypto, where can I check the status of a transaction?
+
+You can use [block explorers](https://ethereum.org/en/developers/docs/data-and-analytics/block-explorers/) to see the status of any transaction in real time. All you need to do is to search your wallet address or the ID of the transaction.
+
+### Can I cancel or return transactions?
+
+No, once a transaction is initiated, you can not cancel the transaction.
```

Comment:
Wondering if it should be made known to the user that an "initiated" tx can actually be cancelled as long as it's still unconfirmed. For example, sending a blank transaction consisting of the same nonce with a higher gas fee.

Alternatively, suggest to change "initiated" to "confirmed" to be more accurate.

Reply:
thank you for your submissions, also agree here. I will update the language

Reply:
Sure thing, let me know if there are any particular pages or write-ups that you might need help on. Trying to see how best to contribute :)